packages/utilities/src/store/slices/user/authApi.ts

```diff
@@ -0,0 +1,92 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+// import { LoginInput } from '../../pages/login.page';
+// import { RegisterInput } from '../../pages/register.page';
+import { GenericResponse, Login } from '@zuri/types';
+import { userApi } from './userApi';
+
+const BASE_URL ='https://api.zuri.chat';
+
+export const authApi = createApi({
+  reducerPath: 'authApi',
+  baseQuery: fetchBaseQuery({
+    baseUrl: `${BASE_URL}/api/auth/`,
+  }),
+  endpoints: (builder) => ({
+    // registerUser: builder.mutation<IGenericResponse, RegisterInput>({
+    //   query(data) {
+    //     return {
+    //       url: 'register',
+    //       method: 'POST',
+    //       body: data,
+    //     };
+    //   },
+    // }),
+    loginUser: builder.mutation< { access_token: string; status: string }, Login>({
```

Comment:
this should be it's own type

the login type is the one that can be placed here as it will almost never be used elsewhere